experimental/examples/bubblebench/editable-shared-tree/src/appState.ts

```diff
@@ -29,24 +29,28 @@ export class AppState implements IAppState {
 
 	public applyEdits() {}
 
-	createInitialClientNode(numBubbles: number): IClient {
-		const client: IClient = {
-			clientId: `${Math.random()}`,
-			color: randomColor(),
-			bubbles: [],
-		};
-
+	createInitialClientNode(numBubbles: number): FlexClient {
```

Comment:
Are IClient and FlexClient interchangeable (structurally equivalent)?

It seems like we're pretty close to being able to use the 'makeClient()' helper defined in 'bubblebench-common' here.

Reply:
No. IClient extends FlexClient though, so I think makeClient could be used here.

For the sake of it being an example, I think its better to use the schema derived types in a realistic way here, and not take advantage of them happening to superset the IClient interface and remove the only decent example of using the FlexTypes.